src/styles/_catalog.scss

```diff
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 200px;
```

Comment:
Create a separate variable for card width, cause you use it in several places, so it makes sense.